fix(docs): barretenberg consent pop up

alejoamiras opened the pull request:
Currently (production, yikes!):

<img width="1292" height="718" alt="image" src="https://github.com/user-attachments/assets/f44e44ee-5008-4692-9ef3-ddb41ed44e62" />

Fix (yay):

<img width="1306" height="894" alt="image" src="https://github.com/user-attachments/assets/4d7cfd59-79ed-4389-b4ef-f6f1d7c771aa" />

critesjosh commented on l1-contracts/lib/circuits:
mistake?

alejoamiras replied:
yes :( always happens to me.

sklppy88 reviewed (approved):
good once removed the unintended submodule change

alejoamiras committed:
fix(docs): barretenberg consent pop up

Currently (production, yikes!):

<img width="1292" height="718" alt="image" src="https://github.com/user-attachments/assets/f44e44ee-5008-4692-9ef3-ddb41ed44e62" />

Fix (yay):

<img width="1306" height="894" alt="image" src="https://github.com/user-attachments/assets/4d7cfd59-79ed-4389-b4ef-f6f1d7c771aa" />

Co-authored-by: Alejo Amiras <alejo.amiras@gmail.com>
